CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,86 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Kolibri is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
@indirectlylit Yes, will be hard to maintain the CoC if it starts having copies in a lot of repos. Let's do as you suggest and change it - I've opted for "Kolibri and KA Lite". The reasoning is more obvious for the part of KA Lite, where LE as an organization should try to steer away from sole ownership and have a contributor from outside of LE still feel included in the community that builds the project! And moreover, that the CoC applies to us all.

Regarding the latter change, I think they intended the phrasing to make it clear that:

1) Diversity is a goal, but it may not always be the reality (hence "possible")
2) Being inclusive to the largest number of contributors is subject to diversity, too, so we don't want to only be inclusive to some majority that would rule out another - so "to be as inclusive as possible" doesn't quite cover. To be really blunt: I'd like to have a CoC that's explicit about offboarding 10 chauvinists to keep 1 minority.

It's all just speculation - I like our diversity :)